.rubocop.yml

```diff
@@ -12,10 +12,15 @@ Lint/EmptyClass:
   Enabled: false
 Naming/PredicateName:
   Enabled: false
+Naming/MethodParameterName:
+  Exclude:
+  - lib/game.rb
+  - lib/game_looper.rb
 Metrics/BlockLength:
   Exclude:
   - spec/game_spec.rb
   - spec/game_looper_spec.rb
+  - spec/ui_spec.rb
```

Comment:
**suggestion (non-blocker)**: You might experiment with changing these excludes to `spec/*` or `spec/**/*`. That should eliminate the need for you to change this file every time you introduce a new spec.